admin/assets/stylesheets/elements/_gds-app-cookie-banner.scss

```diff
@@ -0,0 +1,53 @@
+.app-cookie-banner {
+  font-family: "nta",Arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
```

Comment:
Why is font-size specified twice?

Reply:
@jon-shipley copied from the gds design website. I removed the rem values so there are no longer duplicates.